go-sdk/pkg/sdk/keptn.go

```diff
@@ -303,7 +352,10 @@ func (k *Keptn) createErrorLogEventForTriggeredEvent(triggeredEvent cloudevents.
 
 	errorEventData.Message = err.Message
 
-	keptnContext, _ := triggeredEvent.Context.GetExtension(KeptnContextCEExtension)
+	keptnContext, err2 := triggeredEvent.Context.GetExtension(KeptnContextCEExtension)
+	if err2 != nil {
+		return nil, fmt.Errorf("unable to get extension: %v", err2)
```

Comment:
bad log message... 
maybe something like:
```suggestion
		return nil, fmt.Errorf("unable to get keptn context from '.triggered' event": %v", err2)
```
?